### ISSUE TITLE
the model stopped moving during training but no error

### ISSUE THREAD

**Issue by tqslj2** [mozilla/DeepSpeech#3498]
hi, i'm facing a weired thing that the trainning just stopped move during the first epoch' steps, this is the command that i use to train: 
nohup python3 DeepSpeech.py --alphabet_config_path text.txt --save_checkpoint_dir checkpoint/ --checkpoint_dir checkpoint/ --load_checkpoint_dir checkpoint/ --train_files data/heilongjiang_split_data/train_manifest.csv --dev_files data/heilongjiang_split_data/val_manifest.csv --test_files data/heilongjiang_split_data/test_manifest.csv --audio_sample_rate 8000 --automatic_mixed_precision --export_dir models/ --export_language cmn-Hans-CN --learning_rate 0.001 --train_batch_size 2 --train_cudnn --use_allow_growth --epochs 20 --show_progressbar --reduce_lr_on_plateau --n_hidden &

and this is the nohup.out log file :
Epoch 0 |   Training | Elapsed Time: 0:00:12 | Steps: 104 | Loss: 16.606568
Epoch 0 |   Training | Elapsed Time: 0:00:12 | Steps: 106 | Loss: 16.540244
Epoch 0 |   Training | Elapsed Time: 0:00:12 | Steps: 108 | Loss: 16.420389
Epoch 0 |   Training | Elapsed Time: 0:00:12 | Steps: 110 | Loss: 16.257280

after this step the train have never moved on...

it is weired that neither the memory or the usage of gpu exceeded the limit.

very thankful if anyone have some idea !

**Comment by lissyx** [comment]
Please reach for support on Discourse.
